app/src/main/java/com/keylesspalace/tusky/network/MastodonApi.kt

```diff
@@ -536,8 +536,8 @@ interface MastodonApi {
         @Field("context[]") context: List<String>,
         @Field("irreversible") irreversible: Boolean?,
         @Field("whole_word") wholeWord: Boolean?,
-        @Field("expires_in") expiresIn: Int?
-    ): Call<Filter>
+        @Field("expires_in") expiresInSeconds: Int?
+    ): NetworkResult<Filter>
```

Comment:
all the methods returning `NetworkResult<*>` must be `suspend fun` or they will fail with `NetworkOnMainThreadException`